app/views/user/show.erb

```diff
@@ -1,6 +1,7 @@
 <div class="container">
-  <section class="page-header">
-  <h1><%= profile.username %></h1>
+  <section class="row page-header">
+    <div class="col-md-1 main-profile-img"><img src="https://avatars.githubusercontent.com/u/980783?s=90" /></div>
```

Comment:
@tejasbubane It looks like you hard-coded the image src to your avatar. I think it should be dynamic based on the user that is currently being viewed.